spec/parity/environment_spec.rb

```diff
@@ -31,8 +31,26 @@
     expect(Kernel).to have_received(:system).with(heroku_backup)
   end
 
+  it "correctly connects to the right Heroku app when the local folder's name does not match the Heroku app's name" do
```

Comment:
"finds the correct Heroku app when the $PWD's name does not match the app's name"? It's a little shorter.